src/App.js

```diff
@@ -1,12 +1,122 @@
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+
+    onValue(dbRef, (response) => {
+      const newState = [];
+      const data = response.val();
+      
+      for (let key in data) {
+        newState.push(
+          {
+            key: key,
+            name: data[key]
+          });
+      }
+      setEntry(newState);
+    })
+
+  }, [])
+
+    const handleSubmit = (event) => {
+      event.preventDefault();
+      const database = getDatabase(firebase)
+      const dbRef = ref(database);
+      
+      push(dbRef, userEntry);
+      setUserEntry("");
+      scrollToEntry.current.scrollIntoView({ behavior: 'smooth' })
+    }
+
+    const handleInputChange = (event) => {
+      setUserEntry(event.target.value);
+    }
+
+    const handleBurn = (burnEntry) => {
+      const database = getDatabase(firebase)
+      const dbRef = ref(database, `${burnEntry}`);
+      remove(dbRef);
+    }
+  
   return (
-    <div className="App">
-      <header className="App-header">
+    <div className="App"> 
+      <header className="appHeader">
+        <div className="borderWrapper">
+          <h1><span className="headerSpan1">Morning</span><span className="headerSpan2"> Pages</span></h1>
+        </div>
         
+        <div className="headerTextContainer">
+          <div className="textContainer">
+            <p>An adaptation of a writing exercise from “The Artist’s Way” by Julia Cameron.</p> 
+          </div>
+          <div className="textContainer"> <p> Every morning, write out an unfiltered stream of consciousness in the text box below.</p>
+          </div>
+          <div className="textContainer"><p>Any thought that bubbles up is written down with no judgement.</p></div>
+          <div className="textContainer">
+            <p>The aim is to “empty the mind". The result offers more room for clarity within your headspace.</p>
+          </div>
+          <div className="textContainer">
+            <p>Once your entry is submitted, you may opt to burn your entry.</p>
+          </div>
+        </div>
       </header>
+      <main>
+        <section className="entryForm">
+          <div className="formWrapper">
+            <div className="formContainer">
+              <form className="form"action="submit">
+                <label htmlFor="newEntry">To begin, type your stream of consciousness here:</label>
+              </form>
+            </div>
+            <textarea
+                rows="3"
+                placeholder=""
+                id="newEntry"
+                onChange={ handleInputChange }
+                value={ userEntry }
+                >
+                </textarea>
+            <div className="formButton">
+              <button className="submitEntry" onClick={ handleSubmit } disabled={!userEntry}> Submit my entry </button>
+            </div>
+          </div>
+        </section>
+
+        <section className="burnQueue">
+            <div className="burnHeaderContainer">
```

Comment:
I guess this is more for styling/personal choice -- but you might want to display the h2 above your logged entries li (I thought burn queue was another section that would show the burned entry, and tried to click on it **could also be cause I am a fool lol)